miniapp/USERGUIDE.md

```diff
@@ -163,6 +164,12 @@ val miniAppMessageBridge = object: MiniAppMessageBridge() {
         
         callback.invoke(true, null) // or callback.invoke(false, "error message")
     }
+
+    override fun requestUserName(callback: (isSuccess: Boolean, data: String?) -> Unit) {
+        // requesting user name implementation.
+        // .. .. ..
+        callback.invoke(true, null) // or callback.invoke(false, "error message")
```

Comment:
Looks like the actual implementation of `requestUserName` has `data` as non-nullable. So I think this should be updated to use some example string like `"user_name"`.

Reply:
It looks like iOS is taking the same approach of having a callback: https://github.com/rakutentech/ios-miniapp/pull/149/files#diff-eeb7f9f814d262c7e3820df82237603bR6

But on iOS, they have a `Result` with `success` and `failure` callbacks. Looks like Kotlin also has a `Result` class now, so maybe we should try that? https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-result/#result